### DISCUSSION TITLE
General question regarding callback functions

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#529] [Q&A] [answered]
Hi!
I would like to implement a "click and drag" functionality to select points. I didn't see that option in the vtk docs or the vedo examples. Is there a specific listener event for that or do I have to combine some other events in some way?

Cheers!
j

**Comment by marcomusy** [comment, marked as answer]
Hi 
you may need to create a callback with `Interaction` events:
`vedo -r multiwindows2`

You may also need the `computeWorldPosition()` function, check out:
`vedo -r cutFreeHand`

ps: note that by pressing `a` you toggle actor-mode interaction and you can drag and rotate objects in the scene directly.
